packages/manager/modules/dbaas-logs/package.json

```diff
@@ -12,6 +12,7 @@
   "author": "OVH SAS",
   "main": "./src/index.js",
   "dependencies": {
+    "@ovh-ux/manager-config": "^1.0.0",
```

Comment:
Should be in `peerDependencies` ?

Reply:
@cbourgois It's not an angular module so it's ok to add in it dependencies. Any reason you would rather add it as a peer instead ?

Reply:
Ok, forgot that it's not an angular module. Thanks for reminding me ;)